docs/packaging_policy.md

```diff
@@ -20,3 +25,60 @@ release/download webpage.
 
 - In case of needing those binaries to use them as a "build require" for some library, we will consider following the approach of adding it
   as a system recipe (`<build_require>/system`) and making those binaries available in the CI machines (if the license allows it).
+
+
+## Settings
+
+All recipes should list the four settings `os`, `arch`, `compiler` and `build_type` so Conan will compute a different package ID
+for each combination. There are some particular cases for this general rule:
+ * **Recipes for _header only_ libraries** might omit the `settings` attibute, but in any case they should add
+  
+   ```python
+   def package_id(self):
+      self.info.header_only()
+   ```
+
+ * **Recipes that provide applications** (`b2`, `cmake`, `make`,...) that are generally used as a _build requires_, must list all
+   the settings as well, but they should remove the `compiler` one in the corresponding method unless the recipe provides also
+   libraries that are consumed by other packages:
+
+   ```python
+   def package_id(self):
+      del self.settings.compiler
+   ```
+   
+   Removing the `compiler` setting reduce the number of configurations generated by CI, reducing the time and workload and, at the
+   same time, demonstrates the power of Conan behind the package ID logic.
+
+   > Note.- We retain the `build_type` setting to make it possible for the users to _debug_ these installer packages. We considered
+   > removing this settings and it would be possible to compile these packages in _debug_ mode, but if we remove it from the packageID,
+   > the compiled package would override the existing _release_ binary, and it'd be quite inconvenient for the users to compile the
+   > binary every time they need to switch from _debug_ to _release_.
```

Comment:
I didn't get the meaning of this note the first time I read it, so I am suggesting a full rephrase in case it makes it more clear

```suggestion
   > Note.- Intentionally, the `build_type` setting should not be removed from the package ID in this case. This is done
   to eventually allow users to _debug_ these installer packages.
   > Otherwise, every time installer packages were to be rebuilt switching their build type  between _debug_ and _release_,
   > the compiled package would be overridden and it'd be a quite inconvenient user experience.
```

Reply:
👍  I copied it from the existing FAQ. Now I'm proposing a different paragraph, please, review it again. Thanks!